app.js

```diff
@@ -26,38 +27,73 @@ function getRandomItem(arr) {
     return arr[index];
 }
 
+// getRandomItem(hidingPlaces);
+
 function handleGuess(userGuess, correctSpot) {
-    // should reset the styles
+    // console.log(userGuess);
+    // console.log(correctSpot);
 
+    // should reset the styles
+    resetStyles();
+    
     // then increment the guesses
-
+    totalGuesses++;
+    
     // then grab the appropriate container element for the correct guess from the DOM
+    // const correctContainer = correctSpot + "-container";
+    // console.log(correctContainer);
 
+    const correctContainerEl = document.getElementById(correctSpot + "-container");
+    console.log(correctContainerEl);
+
+    
     // then add the face class to that element so that the face shows up
+    correctContainerEl.classList.add("face");
 
     // then if the user guess is correct, increment the correct guesses
+    if (userGuess === correctSpot){
+        correctGuesses++;
+    };
 
     // update the DOM to show this change to the user (including the losses, not tracked directly in state)
+    totalEl.textContent = totalGuesses;
+    winsEl.textContent = correctGuesses;
+    lossesEl.textContent = totalGuesses - correctGuesses;
 }
 
 function resetStyles() {
     // should remove the face class from all containers
+    shedContainer.classList.remove("face");
```

Comment:
Javascript tends to use single quotes instead of double quotes.  This is what is causing this warning. I'm happy to walk you through how to spot these before you push your code!

Reply:
Thanks!